src/App.tsx

```diff
@@ -1,12 +1,19 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { BrowserRouter as Router } from "react-router-dom";
 import "./App.scss";
 import "antd/dist/antd.less";
 import Header from "./components/Header";
 import Feed from "./components/Feed";
 import ProfileBlock from "./components/ProfileBlock";
+import { wallet } from "./services/wallets/wallet";
+import { useAppSelector } from "./redux/hooks";
 
 const App: React.FC = () => {
+  const walletType = useAppSelector((state) => state.user.walletType);
+  useEffect(() => {
+    if (walletType) wallet(walletType).reload();
+  });
```

Comment:
should wallet be passed in as a dependency?

Reply:
I don't believe so? We want the reload to trigger whenever the site reloads not just when the walletType changes.